src/components/Pagination/hooks/usePaginationQueryParams.js

```diff
@@ -0,0 +1,21 @@
+import { mergeLeft } from "ramda";
+import { useHistory } from "react-router-dom";
+
+import { useQueryParams } from "hooks";
+import { buildUrl } from "utils";
+
+const usePaginationQueryParams = () => {
+  const queryParams = useQueryParams();
+  const history = useHistory();
+
+  const handlePaginationQueryParamsChange = page => {
```

Comment:
In the context of this hook, this function is not an event handler. So, give it a name based on what it does.
```suggestion
  const updatePageInQueryParam = page => {
```